packages/ui-extensions-server-kit/src/ExtensionServerClient/ExtensionServerClient.test.ts

```diff
@@ -7,16 +7,22 @@ const defaultOptions = {
 }
 
 describe('ExtensionServerClient', () => {
+  let socket: WS
+
   function setup(options: ExtensionServer.Options = defaultOptions) {
     if (!options.connection.url) {
       throw new Error('Please set a URL')
     }
-    const socket = new WS(options.connection.url, {jsonProtocol: true})
+    socket = new WS(options.connection.url, {jsonProtocol: true})
     const client = new ExtensionServerClient(options)
 
     return {socket, client, options}
   }
 
+  afterEach(() => {
+    socket.close()
```

Comment:
This was added because I notice that the subsequent tests all fail due to a timeout issue after the first test failure